tests/phpunit/admin/class-wc-stripe-admin-notices-test.php

```
@@ -1023,4 +1075,133 @@ public function test_hide_notices_dismisses_ece_location_notice() {
+
+		$_GET['wc-stripe-hide-notice']     = 'subscription_detached';
+		$_GET['wc-stripe-subscription-id'] = $subscription->get_id();
+		$_GET['_wc_stripe_notice_nonce']   = wp_create_nonce( 'wc_stripe_hide_notices_nonce' );
+
+		$notices = $this->create_admin_notices_instance();
+		$notices->hide_notices();
+
+		$this->assertEquals( 'yes', $subscription->get_meta( '_wc_stripe_subscription_detached_notice_dismissed' ) );
+
+		unset( $_GET['wc-stripe-hide-notice'], $_GET['_wc_stripe_notice_nonce'], $_GET['wc-stripe-subscription-id'] );
+	}
```

PHPMD (2.15.0)

[error] test_hide_notices_dismisses_subscription_detached_notice accesses the super-global variable $_GET. (undefined)

(Superglobals)